addons/mail/static/src/components/dialog/dialog.js

```diff
@@ -65,15 +61,15 @@ export class Dialog extends Component {
      * @param {MouseEvent} ev
      */
     _onClickGlobal(ev) {
-        if (this._componentRef.el && this._componentRef.el.contains(ev.target)) {
+        if (this.refs.component.root.el && this.refs.component.root.el.contains(ev.target)) {
```

Comment:
need guard (everywhere in the diff)

This should crash if doing a global click before children are mounted.